raftLogCheck.js

```diff
@@ -0,0 +1,235 @@
+const http = require('http');
```

Comment:
It would be nice if we can support tls, quite a few of our high profile customer deployments use tls

Reply:
It's actually the internal TLS not the external nginx-configured TLS, but yes some customers use this

Reply:
repd admin does not support https yet, but a bug is open about this: https://scality.atlassian.net/browse/S3C-2612

Reply:
Ah right, I forgot about this.